### DISCUSSION TITLE
quick filter on dashboard calendar view

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#31] [Shipped - Live on production]
After a long week of hard work the dashboard calendar still looks the same, with a tedious list of tasks. It would be wonderful if there could be quick filters applied to the calendar view, for instance: Tasks Pending, Tasks Open, Tasks Finished. Also a choice to list only Maintenance tasks, Repair tasks, etc would be useful. By default it could be set to only Tasks Open and to All types of tasks. Then at the end of the week the view would ideally be empty.
![image](https://user-images.githubusercontent.com/117633901/200265071-8942e72a-f380-4ed9-92fd-031c1cb1cc90.png)

**Comment by YvesR** [comment]
Took a quick look with the the dev team, generally possible and good idea.

**Reply by YvesR** [reply]
Add: Would you also like to address a `issue` to a specific person and this person has `own issues/tasks` to take care of? We might add `responsible user` as optional field.

**Comment by BioMedGPH** [comment]
﻿
﻿
﻿﻿Yes it would be helpful if it was possible to assign a responsible user, and have email notification for that person if a new task is assigned. And maybe a notification email to the main account if the responsible person has marked it as done.

Also very helpful: have a notification to the main user if a clinical user creates a problem report for a device. So I can give some nurses-in-charge a samedis account and then they can report issues themselves, without any sending vague emails.

Verstuurd vanaf mijn iPhone



Verstuurd vanaf mijn iPhone
